src/assets/scss/components/_background-image.scss

```diff
@@ -19,7 +19,7 @@
     width: 100%;
     z-index: -1;
     // filter: url("../images/chris_ui_background_filter.svg#image_overlay");
-    filter: url("#{$image-url}#{$chris-background-filter}"); //$chris-background-filter
+    // filter: url("#{$image-url}#{$chris-background-filter}"); //$chris-background-filter
```

Comment:
@PintoGideon is this ok?

Reply:
Yes @jennydaman , I don't remember why I added the filter. I added it a while ago when I was implementing the dark theme for the UI.